src/panels/developer-tools/logs/developer-tools-logs.ts

```diff
@@ -5,17 +5,31 @@ import {
   css,
   TemplateResult,
   property,
+  query,
+  customElement,
 } from "lit-element";
 
 import { HomeAssistant } from "../../../types";
 import { haStyle } from "../../../resources/styles";
 
 import "../logs/system-log-card";
 import "../logs/error-log-card";
+// tslint:disable-next-line
+import { SystemLogCard } from "../logs/system-log-card";
 
-class HaPanelDevLogs extends LitElement {
+@customElement("developer-tools-logs")
+export class HaPanelDevLogs extends LitElement {
   @property() public hass!: HomeAssistant;
 
+  @query("system-log-card") private systemLog?: SystemLogCard;
+
+  public connectedCallback() {
+    super.connectedCallback();
+    if (this.systemLog) {
+      this.systemLog.fetchData();
```

Comment:
we'll end up calling fetchData twice when we open the page.

Reply:
Depends on the timing but yeah possibly, in my test the element was not defined on first connected

Reply:
that's because of how our router works I think. Can we only call this in `connectedCallback` if we have updated at least once (and so are not going to call `firstUpdated` soon)